app/views/admin/induction_periods/edit.html.erb

```diff
@@ -1,8 +1,11 @@
 <% page_data(
   title: "Edit induction period",
-  error: @induction_period.errors.present?
+  error: @induction_period.errors.present?,
+  backlink_href: admin_teacher_path(@induction_period.teacher)
 ) %>
 
+<%= content_for(:backlink_or_breadcrumb) { govuk_back_link(href: admin_teacher_path(@induction_period.teacher)) } %>
```

Comment:
@peterdavidhamilton - `content_for` is called [in the helper](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/helpers/application_helper.rb#L11-L13) already so I think this is unnecessary.